Scenario: A food company YummyCorps is developing a new kind of tasty, wholesome cereal called CrunchieMunchies.There're 60 calories per serving. The company wants to demonstrate to consumers how healthy CrunchieMunchies is in comparison to other leading brands. The reported calorie amounts for different cereal brands are stored in calories dataframe. 

The task is to prove that CrunchieMunchies cereal is the healthiest choice for consumers.

In [1]:
import numpy as np

In [2]:
calories=[70, 120, 70, 50, 110 , 110, 110, 130, 90, 90, 120, 110, 120, 110, 110, 110, 100, 110, 110, 110, 100, 110, 100, 100, 110, 110, 100, 120, 120, 110, 100, 110, 100, 110, 120, 120, 110, 110, 110, 140, 110, 100, 110, 100, 150, 150, 160, 100, 120, 140, 90, 130, 120, 100, 50, 50, 100, 100, 120, 100, 90, 110, 110, 80, 90, 90, 110, 110, 90, 110, 140, 100, 110, 110, 100, 100, 110]

In [3]:
caloriesv2=np.array(calories)

In [12]:
average_calories=np.mean(caloriesv2)
calories_sorted=np.sort(caloriesv2)
median_calories= np.median(caloriesv2)
calorie_std= np.std(caloriesv2)


print(average_calories)
print(calories_sorted)
print(median_calories)
print(calorie_std)

106.88311688311688
[ 50  50  50  70  70  80  90  90  90  90  90  90  90 100 100 100 100 100
 100 100 100 100 100 100 100 100 100 100 100 100 110 110 110 110 110 110
 110 110 110 110 110 110 110 110 110 110 110 110 110 110 110 110 110 110
 110 110 110 110 110 120 120 120 120 120 120 120 120 120 120 130 130 140
 140 140 150 150 160]
110.0
19.35718533390827


Clearly, CrunchieMunchies is lower than most brands in terms of calories.

In [9]:
nth_percentile= np.percentile(caloriesv2, 4)
print(nth_percentile)
more_calories= np.mean(caloriesv2>60)
print(more_calories)

70.0
0.961038961038961


To be more specific, CrunchieMunchies is at the 4th percentile in terms of calories. It has lower calories than 96% of cereal brands.The low calorie amount is a key avantage of CrunchieMunchies. YummyCorps should incorporate the 'lower calories than 96% of cereal brands" stats in its marketing campaigns to promote CrunchieMunchies.